README.rst

```diff
@@ -68,6 +98,20 @@ directly, but instead be manually inspected. E.g.:
    Note that there isn't a comma in the end of the line. The last argument is
    treated as the reason why a suggestion cannot be automatically applied.
 
+Updating
+--------
+To stay current with codespell developments it is possible to build codespell from GitHub via::
+
+    pip3 install --upgrade git+https://github.com/lucasdemarchi/codespell.git
+
+**Important Notes:**
+* Sometimes installing via `pip` will complain about permissions. If this is the case then run with ::
+
+    pip3 install --user --upgrade git+https://github.com/lucasdemarchi/codespell.git
+
+* It has been reported that after installing from `pip`, codespell can't be located. Please check the $PATH variable to see if `~/.local/bin` is present. If it isn't then add it to your path. 
```

Comment:
double-backticks on inline code in RST format (otherwise it's interpreted as a link)